tests/test_saas_web.py

```
@@ -1222,6 +1222,25 @@ def test_video_content_type_accepted_by_validator(self):
+            return "SUCCESS"
+        scope = {
+            'type': 'http',
+            'method': 'POST',
+            'path': '/shrink',
+            'headers': [(b'x-api-key', 'test-key😀'.encode('utf-8'))],
+        }
+        req = Request(scope)
+        try:
+            res = asyncio.run(saas_web.require_api_key(req, mock_call_next))
+            self.assertEqual(res.status_code, 401)
+        except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)